examples/menu.rs

```diff
@@ -102,3 +129,82 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             }
         });
 }
+
+fn enable_interaction_after_initial_animation(
+    mut commands: Commands,
+    mut reader: EventReader<TweenCompleted>,
+) {
+    for event in reader.iter() {
+        if event.user_data == INIT_TRANSITION_DONE {
+            commands.entity(event.entity).insert(InitTransitionDone);
+        }
+    }
+
+    reader.clear();
+}
+
+#[derive(Component)]
+struct InitTransitionDone;
+
+#[derive(Component, Clone, Copy)]
+enum ButtonLabel {
+    Continue,
+    NewGame,
+    Settings,
+    Quit,
+}
+
+fn interaction(
+    mut commands: Commands,
+    mut interaction_query: Query<
+        (Entity, &Transform, &Interaction, &mut UiColor, &ButtonLabel),
+        (Changed<Interaction>, With<InitTransitionDone>),
+    >,
+) {
+    for (entity, transform, interaction, mut color, button_label) in &mut interaction_query {
+        match *interaction {
+            Interaction::Clicked => {
+                *color = CLICK_COLOR.into();
+
+                match button_label {
+                    ButtonLabel::Continue => {
+                        println!("Continue clicked");
+                    }
+                    ButtonLabel::NewGame => {
+                        println!("NewGame clicked");
+                    }
+                    ButtonLabel::Settings => {
+                        println!("Settings clicked");
+                    }
+                    ButtonLabel::Quit => {
+                        println!("Quit clicked");
+                    }
+                }
+            }
+            Interaction::Hovered => {
+                *color = HOVER_COLOR.into();
+                commands.entity(entity).insert(Animator::new(Tween::new(
+                    EaseFunction::QuadraticIn,
+                    Duration::from_secs_f32(0.2),

```

Comment:
Prefer `Duration::from_millis(200)` which doesn't have rounding errors. I'm in the process of converting all examples and the internal library code to avoid floating point calculations as much as possible, for accuracy reasons.